dicompylercore/dvhcalc.py

```diff
@@ -403,7 +438,8 @@ def get_resampled_lut(index_extents,
     index_extents : list
         Dose grid extents as array indices.
     extents : list
-        Dose grid extents in patient coordinates.
+        Dose grid extents in patient coordinates:
+        [col_pos_min, row_pos_min, col_pos_max, row_pos_max].
```

Comment:
Are we using the nomenclature of patient coords being `x, y`? See line 422: https://github.com/dicompyler/dicompyler-core/pull/285/files#diff-66c143dab66cd8ba7b8f2b0d8f44e36d3cd808010d7279b7e46f75aaff3ef7e0R422
If so, this should be swapped to `[xmin, ymin, xmax, ymax]`.

Reply:
Same with line 452? Should it say `(col, row)`? Just want to make sure the nomenclature is consistent, if we are changing all dose grid coord descriptions to `col, row`.

Reply:
I'll follow through the logic carefully on this - I'm not sure I properly followed the resampled lut path.  After looking again at the code, I'm pretty sure `get_resampled_lut` is not currently correct because it does not use the `x_index` to sample against the pixel spacing (which is real-world linked). But I will go through carefully and make sure naming and comments are correct. I'll also make sure there is a test also combining decubitus and resampling.

Reply:
Yes, I agree - let's just correct the comments for now. We should be able to merge in after that.
Resampling can be put into another PR. Thanks so much for your effort on this.

Reply:
Just for background also, on going through all this, I realized that almost all the code is actually in dose grid coordinates, not patient x, y.  The patient coordinates only come into play when finding which dose points are inside a given contour plane (contours defined as (x, y, z) triplets), and when using structure extents (looking at min and max patient x,y and translating to dose grid indices).